charts/platform-system/templates/flux/cert-manager/helm-release-cert-manager.yaml

```diff
@@ -0,0 +1,29 @@
+{{ if and .Values.apps.certManager.enabled (eq .Values.solution.name "flux") }}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: cert-manager
+  namespace: cert-manager
```

Comment:
I think the `HelmRelease` goes in `flux-system` and use `spec.targetNamespace: {{ .Values.apps.certManager.namespace }}`?

Reply:
done